.gitlab-ci.yml

```diff
@@ -0,0 +1,49 @@
+variables:
+  CI_REGISTRY: docker.io
+  CI_REGISTRY_IMAGE: jibrelnetwork/$CI_PROJECT_NAME
+docker-build-master:
+  image: docker:19.03.8
```

Comment:
Oh, `image` also can be moved to the `default` section

Reply:
fixed, thank you